### PR TITLE
Kubernetes setup

### PR DESCRIPTION
**Pull request by ceddy4395**

### CODE REVIEW

File: kubernetes/create-pod.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Pod
+metadata: 
+        name: so-tags-pod
+spec: 
+        containers:
+        - name: so-tags-container
```
Comment (sarajuhosova) [inline kubernetes/create-pod.yml:7]: We moved this to the `smelly-python` organisation, shouldn’t that be reflected here?

File: kubernetes/deployment.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    matchLabels:
+      app: so-tags
+  template:
+    metadata:
+      labels:
+        app: so-tags
+    spec: 
+      containers: 
+      - name: so-tags
+        image: ghcr.io/sarajuhosova/so-tags:latest
```
Comment (sarajuhosova) [inline kubernetes/deployment.yml]: Here as well

File: kubernetes/create-pod.yml (hunk 1 above)
Comment (wouterpolet) [reply kubernetes/create-pod.yml:7]: @ceddy4395 I'd say so, what do you think?

Comment (ceddy4395) [reply kubernetes/create-pod.yml:7]: yes! Forgot we moved whooops

File: kubernetes/deployment.yml (hunk 2 above)
Comment (ceddy4395) [reply kubernetes/deployment.yml]: correct

File: kubernetes/create-pod.yml (hunk 1 above)
Comment (ceddy4395) [reply kubernetes/create-pod.yml:7]: this name is independent of github btw, so here it doensn't matter

File: kubernetes/deployment.yml (hunk 2 above)
Comment (ceddy4395) [reply kubernetes/deployment.yml]: It is fixed now :)

**Pull request merged**